docs/manage_users.md

```diff
@@ -0,0 +1,131 @@
+# Manage users in Percona Everest
+
+Percona Everest provides user management capabilities, enabling you to securely log in through either the Percona Everest UI or the API.
+
+Local user management involves administering Percona Everest users to ensure secure access to database resources. This encompasses tasks such as creating and deleting users, updating their passwords, etc.
+
+When you install Percona Everest, an `admin` user is automatically created, granting full access to the system.
+
+!!! note alert alert-primary "Note"
+
+    It is recommended that you use the admin account for the initial configuration and subsequently transition to Single Sign-On (SSO) integration.
+
+
+## Overview
+
+All user accounts are stored in the YAML format within the `everest-accounts` Secret in the `everest-system` namespace:
+
+??? example "User accounts: YAML format"
+        
+    ```{.text .no-copy}
+    apiVersion: v1
+    kind: Secret
+    metadata:
+        name: everest-accounts
+        namespace: everest-system
+    data:                    
+      users.yaml: YWRtaW46CiAgcGFzc3dvcmRIYXNoOiBhZG1pbgogIGVuYWJsZWQ6IHRydWUKICBjYXBhYmlsaXRpZXM6CiAgICAtIGxvZ2lu
+    ```
+
+Decoding the base64 encoded value will yield a YAML that appears as follows:
+
+
+??? example "YAML after decoding"
+
+    ```{.text .no-copy}
+    admin:
+        passwordHash: <password>
+    enabled: true
+    capabilities:
+        - login
+    ```
+
+## User management commands
+
+This section provides a list of CLI commands for managing users in Percona Everest.
+
+### Retrieve password
+
+You can retrieve the password for the `admin` user by running the following command:
+
+```sh
+everestctl accounts initial-admin-password
+```
+
+!!! note alert alert-primary "Note"
+
+    The passwords in this Secret are stored as hashes. However, the initial admin user has the password stored as plain text for convenient retrieval later on. We strongly recommend that you update the admin password after installation.
+
+
+
+### Update the password
+
+To update the password for an existing user:
+
+```sh
+everestctl accounts set-password -u <username> --new-password $PASSWORD
```

Comment:
@mayankshah1607 if we don't specify the `--new-password $PASSWORD` flag, users will get prompted through the wizard, correct? 
If so maybe should we advertise the version without the `--new-password $PASSWORD` flag so that users get prompted through the wizard and avoid leaking passwords in their shell's history. WDYT?

Reply:
Yeah, that sounds good.. Let's remove the `--new-password` flag

Reply:
Done